data-serving/data-service/src/model/dictionary.ts

```diff
@@ -0,0 +1,22 @@
+import mongoose from 'mongoose';
+
+/**
+ * Schema for a field that follows a dictionary value format, where some values
+ * are explicitly provided and others are imputed. Examples include the symptoms
+ * and chronic disease fields.
+ */
+export const dictionarySchema = new mongoose.Schema({
+    provided: {
+        type: [String],
+        uniqueItems: true,
```

Comment:
This is a mongoDB thing, but does it work in Mongoose? I can't find any documentation on it. Perhaps consider adding a test for this.

Reply:
ugh good catch... added a test, and implemented my own validator